h/services/search_index.py

```diff
@@ -262,8 +275,8 @@ def _get_annotations_from_es(self, annotation_ids):
 
         for hit in hits:
             updated = hit["_source"].get("updated")
-            updated = isoparse(updated).replace(tzinfo=None) if updated else None
-            hit["_source"]["updated"] = updated
+            if updated:
+                hit["_source"]["updated"] = isoparse(updated).replace(tzinfo=None)
```

Comment:
Deleted annotations come back from Elasticsearch as empty documents: `{}`. This bit of code for transforming the `"updated"` value was changing those deleted annotation documents to `{"updated": None}` which was weird. Refactored slightly to leave them as `{}`.